### ISSUE TITLE
Flaky test(s) in OlMeasurementHandler.test.js

### ISSUE THREAD

**Issue by taulinger** [ldbv-by/bav4-nomigration#349]
```
absoluteOlMeasurementHandler.test.1703910062.js OlMeasurementHandler when activated over olMap looks for measurement-layer and adds the feature for copy on save FAILED
[INFO] 	Expected null to equal Object({ adminId: null, fileId: 'f_lastId' }).

```

**Comment by taulinger** [comment]
Still there 😭:

Expected object to have properties
	    adminId: null
	Error: Expected object to have properties
	    adminId: null
	    at <Jasmine>
	    at /tmp/_karma_webpack_391152/commons.js:228930:57

**Comment by taulinger** [comment]
Hopefully solved with #385
